src/core_modules/capture-core/components/WidgetsChangelog/common/ChangelogFilterBar/ChangelogFilterBar.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import React, { useMemo, useState, useCallback } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core/styles';
+import { spacers } from '@dhis2/ui';
+import type { ChangelogFilterProps, FilterValueType } from './ChangelogFilter.types';
+import { DropdownFilter } from './DropdownFilter';
+import { FIELD_TYPES, FILTER_FIELD } from '../Changelog/Changelog.constants';
+
+const styles = {
+    container: {
+        padding: `0 0 ${spacers.dp8} 0`,
+    },
+};
+
+const getFilterField = (id: string) =>
+    (Object.values(FIELD_TYPES).includes(id) ? FILTER_FIELD.FIELD : FILTER_FIELD.DATA_ELEMENT);
```

Comment:
Currently, we are getting a 400 error when opening the changeLog from the trackedEntity Profile widget and trying to filter or sort by an attribute. I think we need to add `FILTER_FIELD.ATTRIBUTE` and `COLUMN_TO_SORT_BY.ATTRIBUTE` and logic to handle the trackedEntity change log.

https://github.com/user-attachments/assets/10f6b7dd-412e-430f-842b-df4423d0ab9d